README.md

```diff
@@ -107,18 +114,24 @@ dockerImagePluginConfig {
         [
             'dockerfile' : file('src/alpha/Dockerfile'),
             'name'       : 'brightsparklabs/alpha',
-            'tags'       : ['awesome-ant', 'testing'],
+            'tags'       : ['awesome-ant', 'testing']
         ],
         [
-            'dockerfile' : file('src/bravo/Dockerfile'),
+            'dockerfile' : file('src/bravo/bravo.Dockerfile'),
             'repository' : 'brightsparklabs/bravo',
             'target'     : 'dev-mode',
             'buildArgs'  : ['--compress', '--quiet'],
+            'contextDir' : file('./src')
         ],
     ]
     imagesDir = new File('build/images/')
     imageTagDir = new File('build/imageTags/')
     continueOnFailure = true
+    deleteOlderImages = true
+    removeDanglingImages = true
+    privateDockerServer = 'docker.brightsparklabs.com'
+    privateDockerUsername = 'username'
+    privateDockerPassword = 'password'
```

Comment:
I presume this is not what we want. How do you get it to use an env variable? We want to document that variation, not this hardcoded version